README.md

```
@@ -1,15 +1,25 @@
-<h1 align="center">Ultimate ⚡</h1>
+<p align="center">
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)